po/de.po

```diff
@@ -131,36 +129,31 @@ msgstr ""
 "Spielen zu bieten."
 
 #: src/launchers/hgl.vala:7
-#, fuzzy
 msgid "Compatibility tools by Valve for running Windows software on Linux."
-msgstr "Kompatibilitätstool zum Ausführen von Windows-Spielen unter Linux"
+msgstr "Kompatibilitätstool zum Ausführen von Windows-Spielen unter Linux."
 
 #: src/launchers/hgl.vala:19 src/launchers/lutris.vala:15
-#, fuzzy
 msgid ""
 "Variant of Wine's VKD3D which aims to implement the full Direct3D 12 API on "
 "top of Vulkan."
 msgstr ""
 "Fork von Wine's VKD3D, der darauf abzielt, die volle Direct3D 12 API auf "
-"Vulkan zu implementieren"
+"Vulkan zu implementieren."
 
 #: src/launchers/hgl.vala:66 src/launchers/lutris.vala:61
 msgid "Wine build compiled from the official WineHQ sources."
 msgstr "Wine-Build gebaut von den offiziellen WineHQ-Quellen."
 
 #: src/launchers/hgl.vala:69 src/launchers/lutris.vala:64
-#, fuzzy
 msgid "Wine build with the Staging patchset."
 msgstr "Wine gebaut mit dem Staging Patcheset."
 
 #: src/launchers/hgl.vala:72 src/launchers/lutris.vala:67
-#, fuzzy
 msgid "Wine build with the Staging patchset and many other useful patches."
 msgstr ""
 "Wine gebaut mit den Staging Patchesets und mit vielen nützlichen Patches."
 
 #: src/launchers/hgl.vala:90 src/launchers/steam.vala:67
-#, fuzzy
 msgid "Custom Proton build for running Windows games, based on Wine-tkg."
 msgstr ""
 "Benutzerdefiniertes Proton-Build für die Ausführung von Windows-Spielen, "
```

Comment:
This was simplified in the app for clarity, please retranslate this sentence to be simpler. :)